tuf/tuf.go

```diff
@@ -295,19 +295,22 @@ func (rs *repoMan) refreshSnapshot(root *Root, timestamp *Timestamp) (*Snapshot,
 	if previous.Signed.Version > current.Signed.Version {
 		return nil, errRollbackAttack
 	}
+
 	// 3.3.3. The version number of the targets metadata file, and all delegated
-	// targets metadata files (if any), in the previous snapshot metadata file, if
-	// any, MUST be less than or equal to its version number in this snapshot
+	// targets metadata files (if any), in the trusted snapshot metadata file,
+	// if any, MUST be less than or equal to its version number in the new snapshot
 	// metadata file. Furthermore, any targets metadata filename that was listed
-	// in the previous snapshot metadata file, if any, MUST continue to be listed
-	// in this snapshot metadata file.
-	targets, err := rs.repo.timestamp()
+	// in the trusted snapshot metadata file, if any, MUST continue to be listed
```

Comment:
Are we missing the check for 
> any targets metadata filename that was listed in the trusted snapshot metadata file, if any, MUST continue to be listed in the new snapshot metadata file.

?

Reply:
Double checking the implementation of that method, but I believe that's what we're comparing. 

For reference: 

```
cat /var/kolide-corp/launcher.kolide.com-443/launcher-tuf/snapshot.json| jq .

{
...
      "targets": {
        "hashes": {
          "sha256": "oqEi27qknJk06aWNtw7xjKZxt7tA1tl4RrSvsOo31FU=",
          "sha512": "Ex1tv6s0FGzbmIJZn6UdSaLHHlvUghKbk+RhZwl6IeDdadwF931wJqMUFMxS5TQbA39KAteURZil07HMGDSruQ=="
        },
        "length": 726
      },
      "targets/releases": {
        "hashes": {
          "sha256": "MsPKSh5BA2Hhy2fLty851V0ZPyo1b8mWzS6jyjCRnjE=",
          "sha512": "nYnl4PxPJCemYtLXb4yxBg8lYrWddBG2EQNJ9tc5vw6qUjR8FzQZ+KoMgnKYHIQJV8RgQTX7/hd9A/g1vCfNHw=="
        },
        "length": 6599
      }
    },
    "version": 137
  }
}
```

That's the trusted snapshot metadata file, and we're trying to compare the `targets/releases` file listed here with 

```
cat /var/kolide-corp/launcher.kolide.com-443/launcher-tuf/targets/releases.json |jq .
{
...
    },
    "version": 127
  }
}
```

Except that we're comparing the local snapshot against the remote one. 

I'll file an issue with regards to this if we need to do additional work to be compliant with this part of the spec.